sqlmesh/core/engine_adapter/snowflake.py

```diff
@@ -102,6 +102,10 @@ def snowpark(self) -> t.Optional[SnowparkSession]:
             ).getOrCreate()
         return None
 
+    def get_current_catalog(self) -> t.Optional[str]:
+        current_catalog = super().get_current_catalog()
+        return current_catalog.lower() if current_catalog else current_catalog
```

Comment:
this doesn't seem correct, why did you need this?

Reply:
It was to make a test pass, i've fixed the test instead and removed this